idaes/models/unit_models/packed_bed_reactor.py

```diff
@@ -0,0 +1,337 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Standard IDAES PBR model.
+"""
+# Import Pyomo libraries
+from pyomo.environ import Constraint, Var, Reference, Block, Expression
+from pyomo.common.config import ConfigBlock, ConfigValue, In, ListOf
+
+# Import IDAES cores
+from idaes.core import (ControlVolume1DBlock,
+                        declare_process_block_class,
+                        MaterialBalanceType,
+                        EnergyBalanceType,
+                        MomentumBalanceType,
+                        UnitModelBlockData,
+                        useDefault)
+from idaes.core.util.config import (is_physical_parameter_block,
+                                    is_reaction_parameter_block)
+
+from idaes.core.util.misc import add_object_reference
+import idaes.core.util.unit_costing as costing
+from idaes.core.util.constants import Constants as const
+
+__author__ = "Andrew Lee, John Eslick, Alejandro Garciadego, Kanishka Ghosh"
+
+
+@declare_process_block_class("PBR")
```

Comment:
I would recommend `PackedBedReactor` as a better name - we normal err on the side of verbose.